.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "libraries/ms-common"]
+	path = libraries/ms-common
+	url = https://github.com/uw-midsun/ms-common.git
+	branch = fw_lib
```

Comment:
Right now it's a dev branch because master doesn't have a rules.mk, but I'll switch it once we get something merged into master for that.